photon-ml/src/main/scala/com/linkedin/photon/ml/avro/data/ScoreProcessingUtils.scala

```diff
@@ -45,8 +45,15 @@ object ScoreProcessingUtils {
       val uid = Option(scoreAvro.getUid).map(_.toString)
       val label = Option(scoreAvro.getLabel()).map(_.toDouble)
       val ids = scoreAvro.getMetadataMap().asScala.map { case (k, v) => (k.toString, v.toString) }.toMap
+      val idsWithUid =
+        if (uid.isDefined) {
+          ids + (DefaultFieldNames.UID -> uid.get)
+        } else {
+          ids
+        }
+      uid.foreach{ id => }
```

Comment:
Missing space between ```foreach``` and ```{```.

Also, are we forcing evaluation here? Should include a comment that clarifies the purpose.